Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.13-alpine3.21
+LABEL maintainer="dmysamo@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
```

Comment:
There is a typo in the environment variable name. It should be `PYTHONUNBUFFERED` instead of `PYTHOUNNBUFFERED`.